dev/src/field-value.ts

```diff
@@ -171,6 +171,28 @@ export class FieldValue {
    *
    * @param {*} other The value to compare against.
    * @return {boolean} true if this `FieldValue` is equal to the provided value.
+   *
+   * @example
+   * let fieldValues = [
```

Comment:
const?

Reply:
(very optional)

Reply:
For better or for worse, we don't use `const` in any of the other samples. Leaving as is.

Reply:
For better or for worse, we don't use `const` in any of the other samples. Leaving as is.